libs/stokesian_dynamics/device_matrix.hpp

```diff
@@ -0,0 +1,658 @@
+// Copyright (c) 2018 Henri Menke. All rights reserved.
+
+#include <iomanip>
+#include <limits>
+#include <type_traits>
+
+#include "thrust_wrapper.hpp"
+
+#ifdef __CUDACC__
+#include <cublas_v2.h>
+#include <cusolverDn.h>
+#endif
+
+#ifdef __CUDACC__
+#define DEVICE_FUNC __host__ __device__
+#else
+#define DEVICE_FUNC
+#endif
+
+#ifdef __GNUC__
+#define MAYBE_UNUSED __attribute__((unused))
+#else
+#define MAYBE_UNUSED
+#endif
+
+namespace policy {
+
```

Comment:
Please add a comment explaining what "policy" means and what happens in this name space